src/scene/materials/standard-material.js

```diff
@@ -755,12 +756,14 @@ class StandardMaterial extends Material {
 
         this._setParameter('material_gloss', this.gloss);
 
-        if (!this.emissiveMap || this.emissiveTint) {
-            this._setParameter('material_emissive', getUniform('emissive'));
-        }
-        if (this.emissiveIntensity !== 1) {
-            this._setParameter('material_emissiveIntensity', this.emissiveIntensity);
-        }
+        Debug.call(() => {
+            if (this.emissiveMap && this.emissive.r === 0 && this.emissive.g === 0 && this.emissive.b === 0) {
+                Debug.warnOnce(`Emissive map is set but emissive color is black, making the map invisible. Set emissive color to white to make the map visible. Rendering [${DebugGraphics.toString()}]`, this);
+            }
+        });
```

Comment:
I can imagine situations where this warning will trigger for intentional situations - like if I'm lerping emissive up and down over time. That might be annoying in debug.

Reply:
I thought about that as well. Ideally you'd lerp `emissiveIntensity` in that case and not the color